driver/controller.go

```diff
@@ -289,6 +289,13 @@ func (s controllerService) ControllerGetCapabilities(context.Context, *csi.Contr
 					},
 				},
 			},
+			{
```

Comment:
Making `csi.ControllerGetCapabilitiesResponse` can be refactored like this.
Please do so for better maintainability.

```go
	capabilities := []csi.ControllerServiceCapability_RPC_Type{
		csi.ControllerServiceCapability_RPC_CREATE_DELETE_VOLUME,
		csi.ControllerServiceCapability_RPC_GET_CAPACITY,
		csi.ControllerServiceCapability_RPC_EXPAND_VOLUME,
	}
	csiCaps := make([]*csi.ControllerServiceCapability, len(capabilities))
	for i, cap := range capabilities {
		csiCaps[i] = &csi.ControllerServiceCapability{
			// ...
		}
	}
```